vispy/visuals/volume.py

```diff
@@ -34,6 +34,8 @@
 coordinates).
 
 """
+from numpy.typing import ArrayLike
```

Comment:
Can you triple check, I'm pretty sure numpy typing was only added in a semi-recent version. I don't think we set a very strict minimum numpy version for vispy.

Reply:
you're right, this is new in 1.20 - will remove!